recipes/mod_wsgi/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "mod_wsgi" %}
+{% set version = "4.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 64a6a177946b78bb1eb8ecb31fb3aea5f6d306c4eccd247433563ed6afceafe5
+
+build:
+  skip: true  # [py27]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
It looks like this has a console entry point:

https://github.com/GrahamDumpleton/mod_wsgi/blob/develop/setup.py#L603-L604

so that should be included and then tested.

Reply:
for now only included. I wait for the windows build and than resolve the second part.